src/backend/base/langflow/api/v1/files.py

```diff
@@ -30,7 +30,7 @@ def get_flow_id(
     flow = session.get(Flow, flow_id_str)
     if not flow:
         raise HTTPException(status_code=404, detail="Flow not found")
-    if flow.user_id != current_user.id:
+    if flow.user_id != current_user.id and flow.user_id != "":
```

Comment:
there shouldn't be any flow without user_id. How did you create it ? That might be the root of this problem

In case of auto login, all the flows should be mapped to the superuser user. (@ogabrielluiz please correct me if I'm wrong)

Reply:
Missing user_id is from Astra. I'll find out more

Reply:
@nicoloboschi That's correct.

Reply:
I  close this PR since this is not reproducible.

Reply:
@nicoloboschi these flows with empty user_id are from the starter_projects. We need to either make it read-only or save them to a different name automatically by users